build/build.ps1

```diff
@@ -326,10 +328,23 @@
     $this.CopyFiles("$src\Umbraco.Web.UI.NetCore\wwwroot\umbraco\js", "*", "$tmp\WebApp\wwwroot\umbraco\js")
     $this.CopyFiles("$src\Umbraco.Web.UI.NetCore\wwwroot\umbraco\lib", "*", "$tmp\WebApp\wwwroot\umbraco\lib")
     $this.CopyFiles("$src\Umbraco.Web.UI.NetCore\wwwroot\umbraco\views", "*", "$tmp\WebApp\wwwroot\umbraco\views")
+
+
+
+    # Prepare templates
+    Write-Host "Copy template files"
+    $this.CopyFiles("$templates", "*", "$tmp\Templates")
```

Comment:
The bin folder gets copied too. We can exclude it by adding a line below ⬇️

Reply:
Good catch